### PR TITLE
feat: LT-1447 Template visible to exporter field

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**
- Added visible_to_exporter field to templates (which may affect the visible_to_exporter field when generating documents)
- Updated existing tests with the create_letter_template function

### CODE REVIEW

File: letter_templates/models.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ class LetterTemplate(TimestampableModel):
     letter_paragraphs = SortedManyToManyField(PicklistItem)
     case_types = models.ManyToManyField(CaseType, related_name="letter_templates")
     decisions = models.ManyToManyField(Decision, related_name="letter_templates")
+    visible_to_exporter = models.BooleanField(blank=False, null=False)
```
Comment (cstobart) [inline letter_templates/models.py]: `blank=False` and `null=False` are defaults for `BooleanField`

Comment (BenAndrew-Kainos) [reply letter_templates/models.py]: It's always good to be explicit is it not?

Comment (cstobart) [reply letter_templates/models.py]: 👍

Comment (BenAndrew-Kainos) [reply letter_templates/models.py]: 👍

**Pull request merged**